### PR TITLE
Add functionality to save drawing via button click and make post request...

### PR DESCRIPTION
**Pull request by joshuaNewman10**
... to server with canvas drawing data

### CODE REVIEW

File: client/app/room/roomCtrl.js [hunk 1]
```diff
@@ -76,6 +82,34 @@
...
+    //makes a request to our server to store the image in the database
+    $scope.saveCanvas = function() {
+      var drawingData = {
+        username: 'testname123',
+        data: $scope.drawingCanvas.toDataURL()
+      };
+
+      var request = {
+        method: 'POST',
+        url: '/room',
```
Comment (garronmichael) [inline client/app/room/roomCtrl.js]: please change this url to '/room/save'

File: client/app/room/roomCtrl.js [hunk 2]
```diff
@@ -76,6 +82,34 @@
...
+        headers: {
+         'Content-Type': 'json'
+        },
+        data: { data: JSON.stringify(drawingData) }
```
Comment (garronmichael) [inline client/app/room/roomCtrl.js]: data --> cavas

Comment (mcdonnell88) [inline client/app/room/roomCtrl.js]: add roomID

File: client/app/room/roomCtrl.js [hunk 3]
```diff
@@ -76,6 +82,37 @@
       $scope.drawingCanvas = canvasFabric;
     };
 
+    //Function: RoomCtrl.saveCanvas()
+    //This function is called when the user clicks the saveCanvas button
+    //It converts the canvas data into a png image string and then
+    //makes a request to our server to store the image in the database
+    $scope.saveCanvas = function() {
```
Comment (mcdonnell88) [inline client/app/room/roomCtrl.js]: may want to change name of saveCanvas to something like saveRoomData -- tihs way we can have one button that will save everything in the room.

**Pull request merged**